### DISCUSSION TITLE
Using divide-{colour1} and border-{colour2} at the same time?

### DISCUSSION THREAD

**Discussion by calthejuggler** [tailwindlabs/tailwindcss#3494] [Help] [answered]
Hello everybody!

I have a project where I have a list of elements separated by `divide-y divide-gray-200`.
Each of those elements has an active state which includes `border-l-2 border-blue-400`.

I was hoping that this would allow both to co-exist. Unfortunately, it only works on the first element. Any other element has a grey left border:
![image](https://user-images.githubusercontent.com/30095183/106485145-94e52900-64b0-11eb-929e-5f3daa73389a.png)
![image](https://user-images.githubusercontent.com/30095183/106485255-b1816100-64b0-11eb-9949-06c6de66ced4.png)

I also tried it without the `divide-gray-200`, but that gives the border-top the blue colour:
![image](https://user-images.githubusercontent.com/30095183/106485615-163cbb80-64b1-11eb-9761-b972b7dae436.png)

Is it possible to set the two border styles as different colours without custom CSS?

I hope somebody can help me.

Please and thank you in advance!

**Comment by adamwathan** [comment, marked as answer]
Hey! In this case I'd recommend using an extra element:

```html
<ul class="divide-y divide-gray-200">
  <li>
    <div class="border-l-2 border-blue-400">...</div>
  </li>
  ...
</ul>
```

**Reply by calthejuggler** [reply]
Awesome! It worked a treat. Thanks :)
